frontend/packages/helm-plugin/src/topology/helm-data-transformer.ts

```diff
@@ -225,3 +229,7 @@ export const getHelmTopologyDataModel = () => {
     return Promise.resolve(getHelmGraphModelFromMap(helmResourcesMap, resources));
   };
 };
+
+export const getHelmResources = (): WatchK8sResources<any> => {
+  return { secrets: { isList: true, kind: 'Secret', optional: true } };
```

Comment:
you need to pass namespace here as well, because of this you are facing the issue with delayed loading of helm node.
```suggestion
export const getHelmResources = (namespace): WatchK8sResources<any> => {
  return { secrets: { isList: true, kind: 'Secret', namespace, optional: true } };
```

Reply:
Thanks, updated.